packages/buttons/src/containers/ButtonGroupContainer.spec.js

```diff
@@ -76,5 +76,15 @@ describe('ButtonGroupContainer', () => {
         expect(button).toHaveProp('tabIndex', -1);
       });
     });
+
+    it('moves focus to the ButtonGroupView if a button receives focus', () => {
```

Comment:
Active element detection is hard to test within Enzyme 😢 

A common approach that I use is to only mock the expected functionality. The `onFocus` composition that we have on each button is only calling `target.parentNode.focus()`. How focus is handled after that point is a browser concern and not really something that is "testable", especially with jsdom.  I suggest just ensuring that we call the correct method with a mock and calling it a day.

An example is:

```
it('moves focus to the ButtonGroupView if a button receives focus', () => {
      const focusMock = jest.fn();

      findButtons(wrapper)
        .at(0)
        .simulate('focus', { target: { parentNode: { focus: focusMock } } });

      expect(focusMock).toHaveBeenCalled();
    });
```

Reply:
ok that makes sense, will do that 👍